### DISCUSSION TITLE
[Crystools ERROR] GPU不运行了

### DISCUSSION THREAD

**Discussion by lambbigcc** [Comfy-Org/ComfyUI#3598] [Q&A]
[Crystools ERROR] For some reason, pynvml is not working in a laptop with only battery, try to connect and turn on the monitor
[Crystools ERROR] Monitor of GPU is turning off (not on UI!)
请教各位，这个问题怎么解决，感谢！
<img width="868" alt="image" src="https://github.com/comfyanonymous/ComfyUI/assets/19486651/c3d51b76-39e6-428c-abf7-79c4b1919e25">

**Comment by wibur0620** [comment]
不用管
